src/components/dropdown-filter/dropdown-filter.js

```diff
@@ -250,6 +250,11 @@ class DropdownFilter extends Dropdown {
     this.props.create(ev, this);
   }
 
+  handleCreateBlur = () => {
+    this.blockBlur = false;
+    this.setState({ open: false });
```

Comment:
maybe this should call `this.handleBlur` incase it needs to trigger any of the other stuff it does?